CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.13.0] - 2024-10-27
```

Comment:
Just one nit left now — convention is to always leave an `Unreleased` section at the top of the file that future PRs after release can add to.

Reply:
Oh I've just overwritten it in place. Restored. Another reason for it to always be there is to see the accumulated diff to HEAD

Reply:
I tend to also leave in the sub-section for Unreleased so that it's easier for new PR authors to know where to fill in their stuff, so:

```
### Added

### Changed

### Deprecated

### Removed

### Fixed

### Security
```

Reply:
And so empty sections get removed when a new release is pulbished?

Reply:
Exactly right!

Reply:
Added subsections. I see the approaches are varying:
https://github.com/tokio-rs/axum/blob/280d16a61059f57230819a79b15aa12a263e8cca/axum/CHANGELOG.md
https://github.com/tokio-rs/tokio/blob/946401c345d672d357693740bc51f77bc678c5c4/tokio/CHANGELOG.md
https://github.com/chronotope/chrono/blob/7974c60649351553e57727ea02f88b66f8835f0e/CHANGELOG.md

Reply:
that's a good idea to not make folks guess, because it's chang*ed*, fix*ed* ... but secur*ity*

Reply:
Yeah, I think that's one of the things keepachangelog was trying to fix!